packages/frontend/web-ui/src/common/helpers/joinGame.ts

```diff
@@ -0,0 +1,24 @@
+import { buildSerializableResponse } from '../http/helpers/buildSerializableResponse';
+import { JoinGameResponse } from '../http/models/JoinGameResponse';
+import { JoinGameSerializedResponse } from '../http/models/JoinGameSerializedResponse';
+import { httpClient } from '../http/services/HttpService';
+
+export const joinGame = async (
+  token: string | null,
+  gameId: string | null,
+  userId: string | null,
```

Comment:
Why are they nullable fields?

Reply:
The same case as the other PR. I supposed that variables maybe could be null, and I extended to my code. I correct this issue right now.